src/common/harvest/harvest.js

```diff
@@ -107,7 +107,7 @@ export class Harvest extends SharedContext {
     }
 
     const fullUrl = `${url}?${baseParams}${payloadParams}`
-    const gzip = qs.content_encoding === 'gzip'
+    const gzip = !!qs?.attributes?.includes('gzip')
```

Comment:
attributes will be a url-encoded string or could theoretically not exist in the harvest (since the other payloads do not add this).  Otherwise, if another payload adds attributes, the `?.includes` should safeguard against that method not existing

Reply:
(You're thinking of `includes?.()` btw)